src/elmish/package.json

```diff
@@ -16,6 +16,6 @@
   "author": "Eugene Tolmachev",
   "license": "Apache-2.0",
   "devDependencies": {
-    "fable-powerpack": "^1.0.0-narumi-3"
+    "fable-powerpack": "next"
```

Comment:
I think we should keep this one fixed as this is the core library and not a sample.